CMakeLists.txt

```diff
@@ -122,15 +122,23 @@ add_definitions("-DHIP_COMPILER_FLAGS=${HIP_COMPILER_FLAGS}")
 if( MIOPEN_BACKEND STREQUAL "OpenCL")
     set(MIOPEN_BACKEND_OPENCL 1)
     find_package( OpenCL REQUIRED )
-    find_program(MIOPEN_HIP_COMPILER hcc
+    find_program(MIOPEN_HIP_COMPILER clang++
         PATH_SUFFIXES bin
-        PATHS /opt/rocm
+        PATHS /opt/rocm/llvm
```

Comment:
This should probably be `HINTS` instead of `PATHS`.

Reply:
Why `HINTS`?